ui.tests/test-module/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think this is not used anywhere and we can remove it?

Reply:
:+1: